public/js/gogs.js

```diff
@@ -1035,4 +1037,18 @@ $(window).load(function () {
             }
         }).trigger('hashchange');
     }
+
+    // Repo clone url.
+    if ($('#repo-clone-url').length > 0) {
+        switch (localStorage.getItem('repo-clone-protocol')) {
+            case 'ssh':
+                if ($('#repo-clone-ssh').click().length === 0) {
```

Comment:
`.click()` triggers the onClick event and returns the jQuery object for method chaining. The code I wrote is equivalent to the following, but performs better as we don't have to evaulate the selector twice.

```js
if ($('#repo-clone-ssh').length === 0) {
    $('#repo-clone-https').click();
} else {
    $('#repo-clone-ssh').click();
}
```